Tests/Engine/InvokeScriptAnalyzer.tests.ps1

```diff
@@ -567,36 +567,43 @@ Describe "Test -Fix Switch" {
 
 Describe "Test -EnableExit Switch" {
     It "Returns exit code equivalent to number of warnings" {
-        if ($IsCoreCLR) {
-            pwsh -command 'Import-Module PSScriptAnalyzer; Invoke-ScriptAnalyzer -ScriptDefinition gci -EnableExit'
+        if ($IsCoreCLR)
+        {
+            $pwshExe = (Get-Process -Id $PID).Path
         }
-        else {
-            powershell -command 'Invoke-ScriptAnalyzer -ScriptDefinition gci -EnableExit'
+        else
+        {
+            $pwshExe = 'powershell'
         }
+
+        & $pwshExe -Command 'Import-Module PSScriptAnalyzer; Invoke-ScriptAnalyzer -ScriptDefinition gci -EnableExit'
+
         $LASTEXITCODE  | Should -Be 1
     }
 
     Describe "-ReportSummary switch" {
-        $reportSummaryFor1Warning = '*1 rule violation found.    Severity distribution:  Error = 0, Warning = 1, Information = 0*'
-        It "prints the correct report summary using the -NoReportSummary switch" {
-            if ($IsCoreCLR) {
-                $result = pwsh -command 'Import-Module PSScriptAnalyzer; Invoke-Scriptanalyzer -ScriptDefinition gci -ReportSummary'
+        BeforeAll {
+            if ($IsCoreCLR)
+            {
+                $pwshExe = (Get-Process -Id $PID).Path
```

Comment:
Thanks for fixing this, now this test should not fail any more when updating the .Net Core SDK :-)